etc/dockerfiles/stack-build/lts-14.0/Dockerfile

```diff
@@ -5,7 +5,7 @@ LABEL maintainer="manny@fpcomplete.com"
 ARG GHC_VERSION=8.6.5
 ARG LTS_SLUG=lts-14.0
 ARG PID1_VERSION=0.1.2.0
-ARG STACK_VERSION=2.1.3
+ARG STACK_VERSION=2.1.5
```

Comment:
This should wait until after the new release is done.  Also, it should be done by adding a _new_ Dockerfile rather than modifying an existing one, like this: https://github.com/commercialhaskell/stack/blob/7fdb9acea87f0b967424dc8b8f97bbb472dde90b/etc/dockerfiles/stack-build/lts-13.25/Dockerfile